INSTALL.md

```diff
@@ -0,0 +1,128 @@
+
+### Git
+
+* Git must be in the system path.
+
+### MongoDB
+
+* Install MongoDB (Version 2.4 or greater) [http://www.mongodb.org/downloads](http://www.mongodb.org/downloads)
+
+### Grunt-CLI
+
+We propose to install grunt-cli globally via the -g option of npm. To find out more about this option, see [https://npmjs.org/doc/global.html](https://npmjs.org/doc/global.html).:
+
+* (Optional) To define the installation location of global npm packages on Unix-like Systems create a file called `.npmrc` with the following contents in your Home directory:
+
+        prefix=GLOBALPATH
+        umask=077
+
+* Anywhere, invoke `npm install -g grunt-cli` or `sudo npm install -g grunt-cli` if you don't have sufficient user privileges. You can check the installation with `which grunt`. If a location is returned everything is fine.
+* (Optional) If the directory `GLOBALPATH/bin` is not in your path (you can check with `echo $PATH`), you need to add it to the path: In your Home directory, create or edit the file `.profile` and add the following line:
+
+        export PATH=GLOBALPATH/bin/:$PATH
+
+### Shell for `npm install`
+
+* You need a shell (a.k.a. command line window) that is able to access the stuff you just installed.
+
+* For Windows 7 and older:
+   * The shell must be set up for running the Microsoft C++ compiler, like so:
+
+            cmd.exe /E:ON /V:ON /T:0E /K "C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd" /Release /X86
+
+* For Windows 7 or 8:
+   * Use the [Visual Studio Developer Command Prompt](http://msdn.microsoft.com/en-us/library/ms229859.aspx)
+
+## Preparations for use
+
+* Check out this project into some folder (let's call it *REPO* from now on)
+* Open a shell as described above and go to *REPO*
+* Run `npm install`
+
+* If python 3 is the default version, the command line for `npm install` must be adapted to the right python version (2.7)
+
+            npm install --python=/usr/bin/python2
+
+* Start mongodb. If you only plan to use it for development, the default settings are ok. The app as well as the tests use these defaults if you did not change the configuration.
+* (Optional) Instructions for authenticated use are [here](lib/persistence/README.md)
+
+### The built-in wiki
+
+To set up the built-in wiki follow [these instructions](softwerkskammer/lib/wiki/README.md)
+
+### Configuring the server
+
+Configuration for Softwerkskammer and SoCraTes:
+
+* Copy `config/example-winston-config.json` to `config/winston-config.json`, remove the comment in the first line, and adapt the paths if you like.
+
+### Running the server
+
+* Open a shell in *REPO*
+* If your installation is fresh, perform the db initialization script:
+   * Run `node softwerkskammer/initialDBSetup`
+* If your installation is fresh or you changed anything:
+   * Run `npm test` (this creates the CSS files and other static contents and performs jshint checking)
```

Comment:
Fixed in 3d7a81c28d2f91c78291266fb10b15588514cc9b

Btw, I suggest you look into https://www.npmjs.com/package/standard